src/geode/basic/filename.cpp

```diff
@@ -29,13 +29,13 @@ namespace geode
 {
     std::string filename_with_extension( absl::string_view path )
     {
-        ghc::filesystem::path filepath{ to_string( path ) };
+        const ghc::filesystem::path filepath{ to_string( path ) };
         return filepath.filename().string();
     }
 
     std::string filename_without_extension( absl::string_view path )
     {
-        ghc::filesystem::path filepath{ to_string( path ) };
+        const ghc::filesystem::path filepath{ to_string( path ) };
```

Comment:
warning: variable 'filepath' is not initialized [cppcoreguidelines-init-variables]

```suggestion
        const ghc::filesystem::path filepath = 0{ to_string( path ) };
```